jinja2/lexer.py

```diff
@@ -423,6 +421,17 @@ def get_lexer(environment):
     return lexer
 
 
+class OptionalLStrip(tuple):
+    """A special tuple for marking a point in the state that can have
+    lstrip applied.
+    """
+
```

Comment:
Worth adding `__slots__ = ()` so we don’t give up slots optimization from tuple?

Reply:
Yeah, I missed that

Reply:
6b9eb6df5a7804ec4210bf449296aae71eb5cd3e